plugins/out_s3/s3.c

```diff
@@ -991,6 +992,8 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
     void *payload_buf = NULL;
     size_t payload_size = 0;
     size_t preCompress_size = 0;
+    char create_time_string[80];
+    char create_time[80];
```

Comment:
do you need both of these?

Reply:
Yes, because I this struct has two places need create_time and one from chunk->create_time and the other one from m_upload->init_time. In order not to be overwritten, I define two chars.

Reply:
I think you can just use one variable for both. See each of those cases can't both happen in the same function call AFAICT.